core/domain/feedback_jobs_continuous_test.py

```diff
@@ -267,29 +274,34 @@ def test_thread_closed_job_running(self):
             thread_1.put()
 
             # Start job.
```

Comment:
nit: please move this comment below `with`

Reply:
Done